sonar-unit-tests.xsd

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
I like the idea. What is a bit of a thing, is finding a suitably "nice" xml validator that wouldn't cause trouble in Travis. Is it okay if the xml package requires node-gyp during install? Seems quite many ride on top of something of native code.